components/Deck/DeckLandingPage.js

```diff
@@ -237,6 +249,7 @@ class DeckLandingPage extends React.Component {
                                     </Segment>
                                     <Segment attached>
                                         <Header size="small" as="h3">Tags:</Header>
+                                        <div className="sr-only">These decks have been recommended as they are similar to this deck"</div>
```

Comment:
There's a `"` out of place, and I'm not sure about the text here @abijames

Reply:
@kprist fixed